### ISSUE TITLE
Allow configuring number of lines in FirstMethodArgumentLineBreak

### ISSUE THREAD

**Issue by CyborgMaster** [rubocop-hq/rubocop#3393]
I would like to support a slightly different style than enforced by this cop.  Single line methods are ok, two line methods are also ok, but ones you get two three, then every argument should be on its own line.  

``` ruby
# good
method(really_long_parameter_name, really_long_parameter_name)

# good
method(really_long_parameter_name, really_long_parameter_name,
  really_long_parameter_name, really_long_parameter_name)

# bad
method(really_long_parameter_name, really_long_parameter_name,
  really_long_parameter_name, really_long_parameter_name, 
  really_long_parameter_name, really_long_parameter_name)

# good
method(
  really_long_parameter_name, 
  really_long_parameter_name,
  really_long_parameter_name, 
  really_long_parameter_name,
)
```

---
## Expected behavior

No offense 
## Actual behavior

```
test/test_helper.rb:376:8: C: Style/FirstMethodArgumentLineBreak: Add a line break before the first argument of a multi-line method argument list. 
method(really_long_parameter_name, really_long_parameter_name,
       ^^^^^^^^^^^^^^^^^^^^^^^^^^
```
## Steps to reproduce the problem

``` ruby
method(really_long_parameter_name, really_long_parameter_name,
  really_long_parameter_name, really_long_parameter_name)
```

This is extremely important! Providing us with a reliable way to reproduce
a problem will expedite its solution.
## RuboCop version

Include the output of `rubocop -V`:

```
➜  allynova git:(rubocop) ✗ be rubocop -V
0.42.0 (using Parser 2.3.1.2, running on ruby 2.3.1 x86_64-darwin15)
```

**Comment by CyborgMaster** [comment]
This same logic would be great for the other First*LinkBreak cops:

FirstArrayElementLineBreak
FirstHashElementLineBreak
FirstMethodParameterLineBreak

**Comment by bbatsov** [comment]
There hasn't been much activity on this ticket and our Core Team is spread too thin on so many tasks, so I'll just close it for the sake of having a cleaner lists of tasks to focus on. We'd gladly review a PR, but it's unlikely that'd we're going to tackle this ourselves in the foreseeable future.
